src/frontend/components/UI/Sidebar/index.scss

```diff
@@ -209,6 +209,12 @@
   padding: var(--space-2xs) 0;
 }
 
+.isRTL .Sidebar.collapsed .SidebarItemWithSubmenu:hover .SidebarSubmenu,
+.isRTL .Sidebar.collapsed .SidebarItemWithSubmenu:focus-within .SidebarSubmenu {
+  right: var(--sidebar-width);
+  left: auto;
```

Comment:
submenu for settings and stores was floating on the left side of the screen for RTL languages